docs/swagger.json

```diff
@@ -0,0 +1,882 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "description" : "Cineast is vitrivr's content-based multimedia retrieval engine. This is it's API.",
+    "version" : "1.0.0",
+    "title" : "Cineast Api",
+    "termsOfService" : "",
+    "contact" : {
+      "name" : "Cineast Team",
+      "url" : "https://vitrivr.org",
+      "email" : "contact@vitrivr.org"
+    },
+    "license" : {
+      "name" : "Apache 2.0",
+      "url" : "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "host" : "localhost:4567",
+  "basePath" : "/",
+  "tags" : [ {
+    "name" : "api/v1",
+    "description" : "namespace description or what?"

```

Comment:
I think this is the tag description

Reply:
The question here is what we want to have in the description in particular.
See https://github.com/vitrivr/cineast/issues/87